test/sim/items/whiteherb.js

```diff
@@ -24,27 +24,71 @@ describe("White Herb", function () {
 		assert.statStage(wynaut, 'spa', 0);
 	});
 
-	it.skip('should activate after Abilities that boost stats on KOs', function () {
+	it('should activate after Abilities that boost SPA on KOs', function () {
```

Comment:
I don't see much reason to include a ton of each Ability in these tests. Soul-Heart and one of the Moxie copies should be good enough.

Reply:
There isn't if they all function exactly the same (within their respective group). I can prune this to just Moxie and Soul-Heart, and will likely have to make Beast Boost it's own test.